projects/core/src/lib/download-vector-layer/download-vector-layer.component.html

```diff
@@ -0,0 +1,71 @@
+<geoengine-sidenav-header>Download Vector Layer</geoengine-sidenav-header>
+
+<form [formGroup]="form">
+    <mat-dialog-content>
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Area</mat-card-title>
+                <mat-card-subtitle>Specify a rectangular region for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <div fxLayout="row">
+                    <mat-form-field appearance="fill">
+                        <mat-label>min X</mat-label>
+                        <input matInput type="number" formControlName="bboxMinX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max X</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxX" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>min Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMinY" />
+                    </mat-form-field>
+
+                    <mat-form-field appearance="fill">
+                        <mat-label>max Y</mat-label>
+                        <input matInput type="number" formControlName="bboxMaxY" />
+                    </mat-form-field>
+                </div>
+
+                <button mat-raised-button color="accent" (click)="selectBox()" [disabled]="isSelectingBox">Select region on the map</button>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Time</mat-card-title>
+                <mat-card-subtitle>Specify a time selection for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <geoengine-time-interval-input formControlName="timeInterval"></geoengine-time-interval-input>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Resolution</mat-card-title>
+                <mat-card-subtitle>Specify a resolution for the download</mat-card-subtitle>
+            </mat-card-header>
+
+            <mat-card-content>
+                <mat-form-field appearance="fill">
+                    <mat-label>Resolution</mat-label>
```

Comment:
The resolution is 1 by default.
Should't it also change regarding on the zoom level, like the extent?

Reply:
https://github.com/geo-engine/geoengine-ui/pull/722/commits/d409b3c6861e3b38233948a282cf07374e8b62d7